examples/CMakeLists.txt

```diff
@@ -51,5 +51,6 @@ add_subdirectory(record-playback)
 add_subdirectory(motion)
 add_subdirectory(gl)
 add_subdirectory(hdr)
+add_subdirectory(on-chip-calib)
```

Comment:
I think the occ is more of a tool than an example, thoughts?

Reply:
I think it's more an example, not enough for a tool. No command line arguments or special logic, just illustrates the flow of API calls.